psfr/psfr.py

```diff
@@ -419,9 +471,16 @@ def _minimize(x, data, model, variance=None, mask=None, oversampling=1):
         return chi2
 
     init = np.array([0, 0])
-    x = scipy.optimize.minimize(_minimize, init, args=(data, model, variance, mask, oversampling),
-                                bounds=((-5, 5), (-5, 5)), method='Nelder-Mead')
-    return x.x
+
+
+    if optimizer_type == 'NM':
```

Comment:
I would give this option a more meaningful name 'NM' is not self-explanatory. Perhaps set string to 'Nelder-Mead'?